.orchestra/support/revng-container/apt.no-extra

```diff
@@ -0,0 +1,4 @@
+APT::Get::Install-Recommends "false";
+APT::Get::Install-Suggests "false";
+APT::Install-Suggests "0";
```

Comment:
Aren't these defaults?

Reply:
They are, but I'd keep them just to be sure